manifests/bind_event.pp

```diff
@@ -0,0 +1,34 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue with the correct messages
+#
+# === Parameters
+#
+# $hostname::                   Set to localhost for qpid-config operations
+#
+# $ssl_cert::                   SSL cert to use for qpid-config commands
+#
+# $queue::                      Name of the event queue
+#
+# $port::                       Port that qpid is listening on
+define qpid::bind_event(
+  $hostname = 'localhost',
+  $ssl_cert = undef,
+  $queue = 'event_queue',
```

Comment:
It's also fine not to have a default. In this case I think that makes sense since there is nothing that works for most people.